Userland/Libraries/LibCore/ArgsParser.h

```diff
@@ -91,17 +93,35 @@ class ArgsParser {
     void add_option(Vector<size_t>& values, char const* help_string, char const* long_name, char short_name, char const* value_name, char separator = ',', OptionHideMode hide_mode = OptionHideMode::None);
 
     void add_positional_argument(Arg&&);
-    void add_positional_argument(char const*& value, char const* help_string, char const* name, Required required = Required::Yes);
-    void add_positional_argument(String& value, char const* help_string, char const* name, Required required = Required::Yes);
-    void add_positional_argument(StringView& value, char const* help_string, char const* name, Required required = Required::Yes);
+
+    template<typename T>
+    requires(IsAppendableContainer<T>) void add_positional_argument(T& values, char const* help_string, char const* name, Required required = Required::Yes)
+    {
+        add_positional_argument_helper<true>(help_string, name, required,
+            [&values](char const* s) {
+                values.append(s);
+                return true;
+            });
+    }
+
+    template<typename T>
+    requires(IsOneOf<T, char const*, String, StringView>) void add_positional_argument(T& value, char const* help_string, char const* name, Required required = Required::Yes)
```

Comment:
You can collapse this down to
```suggestion
    template<OneOf<char const*, String, StringView> T>
    void add_positional_argument(T& value, char const* help_string, char const* name, Required required = Required::Yes)
```

Reply:
Very neat!